### DISCUSSION TITLE
Errors when trying to build any example

### DISCUSSION THREAD

**Discussion by LucHarding** [hathach/tinyusb#1754] [Q&A]
I have cloned the TinyUsb repo, following the instructions on the getting started page, but when I try to build any of the examples, I get this error:

![image](https://user-images.githubusercontent.com/118918474/203589297-a428861e-f15a-402a-9f8b-91ba23b10aed.png)

This error happens whenever I try and build any of the examples for any of the boards, anyone know why this is happening? I have tried using GNU Make 4.3 installed though chocolatey, and also mingw32-make from winlabs and both produce the same error.

**Comment by hathach** [comment]
try to search this dicussion + issue, there is other user tips for compiling on windows.

**Comment by LucHarding** [comment]
I managed to build the examples after some efforts, I read through [this](https://github.com/hathach/tinyusb/discussions/1030) discussion and found a suggestion at the bottom for using gccvar.bat.
